packages/vscode-extension/src/project/deviceSession.ts

```diff
@@ -262,13 +276,10 @@ export class DeviceSession implements Disposable {
     return previewUrl;
   }
 
-  private async startDebugger() {
-    if (this.debugSession) {
-      this.debugSession.dispose();
-    }
-    this.debugSession = new DebugSession(this.metro, this.debugEventDelegate);
-    const started = await this.debugSession.start();
-    if (started) {
+  private async connectDebugger() {
```

Comment:
Lets be consistent and use `JSDebugger` here. We have too many things called debugger or debug session already.